### PR TITLE
 Update ChatQnA README.md

### PR DESCRIPTION
**Pull request by xiguiw**
Add pull docker images from docker hub

## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Issues

https://github.com/opea-project/GenAIExamples/issues/412

## Type of change

Document update

- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

No

## Tests

Verified pull down docker images works as expected.

### CODE REVIEW

File: ChatQnA/docker/xeon/README.md [hunk 1]
```diff
@@ -57,6 +57,34 @@ chaqna-xeon-ui-server
 Port 5173 - Open to 0.0.0.0/0
 ```
 
+You can pull ChatQnA and components docker images form docker hub or build images from source code.
+
+## 🚀 Pull docker images
```
Comment (chensuyue) [inline ChatQnA/docker/xeon/README.md:62]: No need to pull the image manually, when you run docker compose, the image will be pull automatically.

Comment (arun-gupta) [reply ChatQnA/docker/xeon/README.md:62]: Exactly, there is no need to pull the Docker images explicitly at all.

File: ChatQnA/docker/xeon/README.md [hunk 2]
```diff
@@ -57,6 +57,34 @@ chaqna-xeon-ui-server
...
+### Use vLLM as backend
+
+```
+docker pull opea/vllm:latest
+docker pull opea/llm-vllm:latest
+```
+
+Go to [Section Check Docker Images)](#Check-Docker-Images)
+
 ## 🚀 Build Docker Images
```
Comment (arun-gupta) [inline ChatQnA/docker/xeon/README.md:88]: This section should be moved to a backup section. Keep the flow simple for developer to get started.

**Review by arun-gupta — COMMENTED** [review]
The flow of instructions can be further simplified.

File: ChatQnA/docker/xeon/README.md (hunk 1 above)
Comment (xiguiw) [reply ChatQnA/docker/xeon/README.md:62]: Hold on this change as some images are behind in terms of approval

**Pull request closed without merging**